frontend/testing/playwright/tests/org-library/codelists.spec.ts

```diff
@@ -57,34 +57,19 @@ test('that it is possible to create a new codelist', async ({ page, testAppName
   // This situation might happen if a test is cancelled right after the creation of the code list.
   if (codeListTitleAlreadyExists) {
     await orgLibraryPage.codeLists.clickOnCodeListAccordion(CODELIST_TITLE_MANUALLY);
-    await deleteAndVerifyDeletionOfCodeList(orgLibraryPage, CODELIST_TITLE_MANUALLY);
+    await deleteAndVerifyDeletionOfOpenedCodeList(orgLibraryPage, CODELIST_TITLE_MANUALLY);
   }
 
   await orgLibraryPage.codeLists.clickOnCreateNewCodelistButton();
   await orgLibraryPage.codeLists.verifyNewCodelistModalIsOpen();
   await orgLibraryPage.codeLists.writeCodelistTitle(CODELIST_TITLE_MANUALLY);
 
-  await orgLibraryPage.codeLists.clickOnAddAlternativeButton();
-
-  await orgLibraryPage.codeLists.verifyNewItemValueFieldIsVisible(
-    EXPECTED_NUMBER_OF_ITEMS_IN_MANUALLY_CREATED_CODELIST,
-  );
-  const firstRowValue: string = 'First value';
-  await orgLibraryPage.codeLists.writeCodelistValue(
-    EXPECTED_NUMBER_OF_ITEMS_IN_MANUALLY_CREATED_CODELIST,
-    firstRowValue,
-  );
-  const firstRowLabel: string = 'First label';
-  await orgLibraryPage.codeLists.writeCodelistLabel(
-    EXPECTED_NUMBER_OF_ITEMS_IN_MANUALLY_CREATED_CODELIST,
-    firstRowLabel,
-  );
```

Comment:
These checks won't work with the new interface. I removed them completely since they are also redundant (there is another test for adding a code item).

Reply:
Great that you comment the reason! 🚀